CI/config/common_test_config.yaml

```diff
@@ -0,0 +1,31 @@
+kraken:
```

Comment:
@paigerube14 thoughts on add docs around modifying the CI configs in case we add in a few field?

Reply:
Added in 2 docs
* https://github.com/paigerube14/kraken/tree/ci_tests/CI
* https://github.com/paigerube14/kraken/blob/ci_tests/docs/test_your_changes.md

Please review and let me know if there's anything I should change